helm/cas-metabase/values.yaml

```diff
@@ -44,7 +44,7 @@ metabase:
       cpu: 1
       memory: 4Gi
     requests:
-      cpu: 50m
+      cpu: 250m
```

Comment:
If we don't increase the CPU requests, then we'll actually end up having more resources available, as we can fit more pods in our namespace quota